SAVEMake.defs

```diff
@@ -0,0 +1 @@
+/home/hujun5/downloads1/vela_sim/nuttx/tools/../boards/sim/sim/sim/scripts/Make.defs
```

Comment:
Please fix it, use are using your local absolute path instead of NuttX relative path